.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "cpp/third_party/objecthash"]
```

Comment:
In `cpp`? It contains other versions, and seems to me that a `third_party` is a generic thing?